docs/code_editor/custom_response_code.mdx

```diff
@@ -0,0 +1,5 @@
+# Custom Response Code
+
+For all sync run jobs endpoints, if the response contains a key `windmill_status_code` with a number value, that value will be used as the status code.
+
```

Comment:
This should be mentionned somewhere in core concepts and this is more of a script editor concept